conf/bk_server.conf

```diff
@@ -891,6 +891,11 @@ zkEnableSecurity=false
 # bookie then it would throw BKNotEnoughBookiesException instead of picking random one.
 # enforceMinNumRacksPerWriteQuorum=false
 
+# Enforce write being acknowledged by bookies belonging to atleast minimum
+# number of fault domains(depending on the placement policy) before being
+# acknowledged by bookkeeper.
+# enforceMinNumFaultDomainsForWrite=true
```

Comment:
Considering https://github.com/apache/bookkeeper/blob/master/conf/bk_server.conf#L887 and `minNumRacksPerWriteQuorum` is not set by default, this would be incorrect to do.

Reply:
sorry, this value should be set to false by default.

Reply:
Modifying.